sdks/java/container/build.gradle

```diff
@@ -47,7 +47,7 @@ task copyDockerfileDependencies(type: Copy) {
   from configurations.dockerDependency
   rename "slf4j-api.*", "slf4j-api.jar"
   rename "slf4j-jdk14.*", "slf4j-jdk14.jar"
-  rename "harness.*\\.jar", "beam-sdks-java-harness.jar"
+  rename '(beam-sdks-java-harness)-(.*).jar', '$1.jar'
```

Comment:
I would leave this as:
`rename 'beam-sdks-java-harness-.*.jar', 'beam-sdks-java-harness.jar'`

Because that is what the boot.go code expects and if the name of the output changes, we don't want to have to change boot.go as well.